### DISCUSSION TITLE
Diff viewer broken in Firefox when expanding

### DISCUSSION THREAD

**Discussion by lminiero** [community/community#5958] [Projects and Issues]
For a while, viewing diffs in Firefox seems broken. This seems to happen when using tabs for indentation, and expanding up/down. As you can see in the picture below, the indentation for the expanded part doesn't match the rest:

![firefox-github](https://user-images.githubusercontent.com/3684796/135884112-dad13a9e-fa37-4c96-90d2-bb8cf5ed3c9d.png)

It looks like an issue handling whitespaces, but inspecting the elements I couldn't figure out if it's a problem of GitHub, or a bug in Firefox. This works as expected in Chrome instead.

**Comment by m1kola** [comment]
I see this every day in a project where we use tabs (Go code) for indents. Happens on both unified an split diffs. I use Firefox 94.0.1. on MacOS 11.5.2.

Works fine in Chrome 95.0.4638.69 (Official Build) (x86_64) and in Safari 14.1.2 (16611.3.10.1.6).

![image](https://user-images.githubusercontent.com/509198/142026549-84cd9878-2d89-4b51-affb-bbc60fd69cea.png)

**Comment by indradhanush** [comment]
I have the same problem on Firefox and my version is:

```
➜ firefox --version
Mozilla Firefox 94.0.2
```


![image](https://user-images.githubusercontent.com/2682729/145536498-1382999d-6553-4526-8ab0-390858839a24.png)

**Comment by kaworu** [comment]
Same issue here, really hope this get fixed.
```
% firefox --version
Mozilla Firefox 100.0
```

![Screenshot 2022-05-06 at 13-06-44 vtep Documents VTEP redirection and L7 policy partial incompatibility by vincentmli · Pull Request #19700 · cilium_cilium](https://user-images.githubusercontent.com/109415/167121563-6a24d7da-de2f-4246-a231-eb56469a1dba.png)

**Comment by Exxion** [comment]
This is still broken, and mysteriously seems related to the font attributes. Disabling the `font-family` and `font-size` attributes makes all the whitespace line up. Of course, it also removes the monospace font, so it's not exactly a solution.
The wider tabs are the correct ones. The narrower ones are half the width they should be.
All of the attributes mentioned here plus `tab-width` are identical between the two cases, but they still render differently. I have no idea why.

Mozilla Firefox 101.0

**Comment by MathiasReker** [comment]
Try using another browser

**Comment by flame-0** [comment]
I can reproduce this too. Works fine in chromium-based browser (ungoogled-chromium)
